site/en/docs/webstore/spam-faq/index.md

```diff
@@ -2,156 +2,202 @@
 
-Only one non-production build is allowed to be 'published to public'. The rest, if any, should be
-published to trusted testers.
 
-### Can developers appeal a repetitive content violation? {: #appeal }
+
+### Can developers appeal a repetitive content violation?
 
 You can appeal a repetitive content violation if you think it was flagged in error. To appeal, see
 the instructions included in the notification email—this is sent to the email address listed under
 your CWS developer account.
 
-### How can developers consolidate their user base? {: #consolidation }
+
+### How can developers consolidate their user base?
 
 Developers can only consolidate their user base before enforcement action is taken against their
 extensions, beginning on August 27, 2020. If you do not remove the repetitive extensions before this
-policy is enforced against them, you will lose your user base.
+policy is enforced against them, you will lose your user base. 
 
 If you have repetitively submitted extensions and want to retain your user base:
 
-- Choose one instance to retain
-- Update the other instances to tell your users how to migrate to the retained instance
-- Publish these updates
-- Remove all other instances from the Chrome Web Store in order to avoid enforcement action against
-  your extensions.
 
-### Will any of the repetitive extensions be retained? {: #retention }
+
+* Choose one instance to retain
+* Update the other instances to tell your users how to migrate to the retained instance
+* Publish these updates
+* Remove all other instances from the Chrome Web Store in order to avoid enforcement action
+  against your extensions. 
+
+
+### Will any of the repetitive extensions be retained?
 
 No. On August 27, 2020, the Chrome Web Store will begin removing repetitively submitted extensions.
 None of them will be retained. This is why you should make sure to bring your listings into
 compliance and remove duplicates before August 27, 2020.
 
-### Are extensions that are language specific (differing only in localization) considered duplicate? {: #localization }
+Will extensions that provide duplicate experiences, but that are geo-focused & localised (language
+specific) be considered a violation of repetitive content policy? For example:
 
-Yes. Extensions that provide duplicate experiences—and which differ only in localization—violate
-the policy. For example:
+ - Product ABC - UK
+ - Product ABC - Germany
 
-- Product ABC - UK
-- Product ABC - Germany
+Yes. Extensions that provide duplicate experiences—and which differ only in localization—violate the
+policy. Developers should implement any language-specific code within a single extension.
 
-Developers should implement any language-specific code within a single extension.
 
-## **Keyword spam** {: #keyword-spam }
+<h2>Keyword spam</h2>
 
-### How many sites/keywords/brands can be listed in the metadata? {: #keyword-repitition }
+### How many sites/keywords/brands can be listed in the metadata?
```

Comment:
```suggestion
### How many sites/keywords/brands can be listed in the metadata? {: #keyword-repetition }
```

Re-add the ID tag. Unfortunately there was a typo in the previous tag name. The original motivation for adding these tags was to make the final URL more human readable and consistent across title changes. Compare the following:

* Explicit: https://developer.chrome.com/docs/webstore/spam-faq/#keyword-repetition
* Auto-generated: https://developer.chrome.com/docs/webstore/spam-faq/#how-many-siteskeywordsbrands-can-be-listed-in-the-metadata